packages/react-charting/etc/react-charting.api.md

```diff
@@ -787,13 +787,14 @@ export interface ILegendsProps {
     canSelectMultipleLegends?: boolean;
     centerLegends?: boolean;
     className?: string;
+    defaultSelectedLegends?: string[];
     enabledWrapLines?: boolean;
     focusZonePropsInHoverCard?: IFocusZoneProps;
     legends: ILegend[];
+    onChange?: (selectedLegends: string[]) => void;
     onLegendHoverCardLeave?: VoidFunction;
     overflowProps?: Partial<IOverflowSetProps>;
     overflowText?: string;
-    selectedLegend?: string;
```

Comment:
Do not alter existing public interface. You can add new ones without breaking existing consumers.

Reply:
this property `selectedLegend` has not been implemented yet.
I was misled by this API.

So, I think can remove it.

Reply:
Sounds good.